internal/executor/executor.go

```diff
@@ -55,6 +64,12 @@ func (e *RealExecutor) Execute(output io.Writer, workingDir string, name string,
 }
 
 func (e *RealExecutor) ExecuteAndCapture(output io.Writer, workingDir string, name string, args ...string) (string, error) {
+	if flags.DryRun {
+		msg := fmt.Sprintf("Dry-run mode: %s %s. Working dir: %s", name, args, workingDir)
```

Comment:
Same as comment above in `Execute`

Reply:
Done in https://github.com/Skyscanner/turbolift/pull/88/commits/89a68e308ef04702b6a1a0859764d960e9a75a53